### PR TITLE
Page with css

### PR DESCRIPTION
**Pull request by gabriel9216**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+function check(number) {
+    if(number=== "" || isNaN(number)) {
```
Comment (niedzielnyaniol) [inline script.js]:
```suggestion
    if(number=== "" || typeof number !== "number") {
```

`typeof` is more common. `isNaN` also do a job but is very rare

File: script.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+function check(number) {
```
Comment (niedzielnyaniol) [inline script.js]: `checkIsNumber`. Check is too generic to me

File: script.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    if(number=== "" || isNaN(number)) {
+        alert("Prosze wpisać wartość liczbowa");
+        return false;
+    } else
+    return true;
+}
+function celcjusz()
+{
+	let number=document.getElementById('number').value;
+	let celcjusz=number*'1.8'+'32';
```
Comment (niedzielnyaniol) [inline script.js]: why do you use strings `'1.8'` etc instead of numeric number `1.8`?
It looks hacky

File: script.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	let result=document.getElementById('result');
+	if(check(number)){
+		result.innerHTML=number+" stopnie celcjusza to "+celcjusz+" fehrenheitów";
+	}
+
+}
+function fahrenheit()
+{
+	let number=document.getElementById('number').value;
+	let fahrenheit=(number-'32')/'1.8';
```
Comment (niedzielnyaniol) [inline script.js]: strings instead of numbers

File: script.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+	let celcjusz=number*'1.8'+'32';
+	let result=document.getElementById('result');
+	if(check(number)){
+		result.innerHTML=number+" stopnie celcjusza to "+celcjusz+" fehrenheitów";
+	}
+
+}
+function fahrenheit()
+{
+	let number=document.getElementById('number').value;
```
Comment (niedzielnyaniol) [inline script.js]: you use `document.getElementById('number').value` second time.

Try to create function that returns this and then use it like that

```suggestion
	let number= getInputValue()
```

File: script.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	let fahrenheit=(number-'32')/'1.8';
+	let result=document.getElementById('result');
```
Comment (niedzielnyaniol) [inline script.js]:
```suggestion
	let result=getResult();
```

File: styl.css [hunk 7]
```diff
@@ -0,0 +1,29 @@
+*{
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+body{
+    background-color: aqua;
+}
+h1{
+    font-size: 150%;
```
Comment (niedzielnyaniol) [inline styl.css]: do not use % in font size. Use `px` or `rem` Its more common

File: styl.css [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    color: black;
+    text-align: center;
+}
+button{
+    background-color: blue;
+    color: white;
```
Comment (niedzielnyaniol) [inline styl.css]: try to use hex `#fff` instead of name of color - its kinda standard

**Comment by niedzielnyaniol** [comment]
image
you have conflict in this PR. You need to resolve it first in this `page_css` branch. Here is random video how to do it https://www.youtube.com/watch?v=mxDN0rYQyGA&ab_channel=overment.

When you solve it - merge it

File: styl.css (hunk 8 above)
Comment (gabriel9216) [reply styl.css]: done

File: styl.css (hunk 7 above)
Comment (gabriel9216) [reply styl.css]: done

File: script.js (hunk 3 above)
Comment (gabriel9216) [reply script.js]: done

File: script.js (hunk 2 above)
Comment (gabriel9216) [reply script.js]: done

**Pull request merged**